components/sankey/elements/title.md

```diff
@@ -26,13 +26,13 @@ The `<SankeyTitle>` tag exposes the following parameters for customization of th
 
 | Parameter | Type and Default&nbsp;Value | Description |
 | --------- | ---- | ----------- |
-| `Align` | `SankeyTitleAlign` enum | The alignment of the title. |
+| `Align` | [`SankeyTitleAlign` enum](/blazor-ui/api/telerik.blazor.sankeytitlealign) <br/> (`SankeyTitleAlign.Left`) | The alignment of the title. |
 | `Background` | `string`  | The background color of the title. |
-| `Color` | `string`  | The text color of the title. |
+| `Color` | `string` <br/> (`rgb(66, 66, 66)`)  | The text color of the title. |
```

Comment:
Somewhere else I saw a default value in HEX format. Is the documentation inconsistent, or do we really set the same color in two different ways?

Reply:
You are probably referring to the [links color](https://docs.telerik.com/blazor-ui/components/sankey/elements/links#basic-customization) but the default color seems to be indeed set in HEX.